.gitattributes

```diff
@@ -0,0 +1,2 @@
+# .gitattributes
+* text=auto eol=lf
```

Comment:
Great job. As Windows user. Sometimes it always goes CRLF even I don't want to. This is good way to force git clone to use LF only